src/socket.io/posts/tools.js

```diff
@@ -174,12 +174,17 @@ module.exports = function (SocketPosts) {
 
 		var postData = await posts.changeOwner(data.pids, data.toUid);
 		postData = postData.map(({ pid, tid, uid, cid, timestamp }) => ({ pid, tid, uid, cid, timestamp }));
-		await events.log({
-			type: 'post-change-owner',
-			uid: socket.uid,
-			ip: socket.ip,
-			newOwner: data.toUid,
-			originalPosts: postData,
+		var promises = [];
+		postData.forEach((post) => {
```

Comment:
Suggest using map here instead

Reply:
Yeah, I think I should stop programming after being awake for 18-19 hours :D I already was doing a map there so there was no reason to have an additional loop there. Thanks.